internal/server/http/handler_test.go

```diff
@@ -72,41 +78,3 @@ func TestImposterHandler(t *testing.T) {
 		})
 	}
 }
-
-func TestInvalidRequestWithSchema(t *testing.T) {
```

Comment:
This test has been removed because tests present on this file only test the `ImposterHandler` code, so this test has no relationship with the code tested at all (i.e. you can change the request as much as you want that it never won't break this test).